### Using tombo to detect methylation from ONT data

Datatest is the GMI1000 genome (Ralstonia solanacearum). Dataset has a very high coverage (10,000X). We first tested Tombo on randomly selected 10% of data.

We tested our data against all 4 models available in Tombo package : 5mC, 6mA, dam, dcm

Despite working on a small sample of the sequenced reads, more than 99% of the adenosine and cytosine were rightly mapped on our reference genome. Coverage is close to 1000x for most sites.

Interestingly the 1% of adenine and cytosine that were not found in Tombo results is for the majority located in the same region of the plasmide (1490000 - 1495000), both on strand + and -. That is showing this 5000pb region was probably not covered by any reads thus not sequenced (at least it's what we see in the 10% sample of the whole data). This sequence may be missing from the strain.

Distribution of modified bases fractions show consistency between the chromosome and plasmide of r.solanacearum for every model. Histogram of fraction distributions for 5mC, 6mA, dcm show a majority of fraction corresponding to non-methylated bases (0 to 0.4). In an other way, adenine contained in dam motifs show an almost linear distribution with its highest peak around fractions of value 1. This indicates that the proportion of methylated bases is found higher for adenine in dam context than in general contexts.

Tombo models 5mC and 6mA look for any possible modified A or C in any context (motifs). It means that A or C that are contained in dam or dcm contexts will also be tested against 6mA and 5mC.In this optic we looked for correlation of shared mapped bases between 5mC dcm and between 6mA dam. We saw that correlation is 10% greater between 5mC and dcm (0.7) than betwwen 6mA and dam (0.6).

To assess the resulted fractions of methylated reads at genomic we compared our methylated calls with pacbio data. We found general poor correlation between fraction at each shared genomics positions.

We also compare one normal strain to a mutant methyltransferase one. Found very high correlation.

On controlled data : Several program have been developped to assess the accuracy of modified nucleotides detections mcaller : using orthogonally validated positions (use of P. aeruginosa as negative control). "Across species, calculations of percent 6mA detected as methylated by PacBio correlated highly (0.97), further increasing confidence that most motifs were correcly identified" "Adenine marked as methylated by PacBio that did not correspond to motifs were caled methylated by mcaller only 10/20% cases" --&gt; without motif.

On model based algo:

Tombo : no model training require but sequencing of native and matched amplified DNA sample. Model-based included since v1.5 as one of the three detection methods
